spec/osl_gitlfs_spec.rb

```diff
@@ -53,11 +53,17 @@
       end
       it do
         expect(chef_run).to nothing_execute('git lfs pull /foo').with(
+          user: 'root',
+          group: 'root',
+          login: true,
           cwd: '/foo'
         )
```

Comment:
Would it be worth checking that a non-default user is set correctly in this spec?

Reply:
I adjusted the tests so we do both a unit and integration test with a non-root user.